### PR TITLE
Update rules.md

### PR DESCRIPTION
**Pull request by antifoo**

### CODE REVIEW

File: admin/rules.md [hunk 1]
```diff
@@ -313,8 +313,12 @@ layout: default
     2.  A member of the management committee must be a fully paid member of the
         association.
     3.  At each annual general meeting of the association, the members of the
-        management committee must retire from office, but are eligible, on
-        nomination, for re-election.
+        management committee agree to retire from office at a date no more than 
+	two months after the AGM.
+	During this time the outgoing management committee members will:
```
Comment (jhogendorn) [inline admin/rules.md:318]: The thing with the rules is that they are rules, not guidelines or shoulds or procedure etc. Putting in soft things like these dotpoints is not a problem, but we've rarely found it to add value.

What this section needs is more clear rules about how handover happens, ie for 2 months you have 2 exec, who's actually in charge? are there milestones of handover? 

for example, does the old treasurer only step down once the audit is complete? secretary once the return of association is submitted? that would be better for people leaving big jobs for the next, and helps train new people on that responsibility.

It's taken us 12 months and we still havent solved all handover issues, i'm not sure how to solve handover when its every year and it takes that long to do. we can get better at it but some things are incredibly painful.

**Comment by davidbuzz** [comment]
I have seen other grps have a 12mth handover, so exec are 'incoming' for a year, 'current' for a year, 'outgoing' for a year.. but i suspect that a 3yr commiment is overkill fir hsbne. ..  but maybe the overlapping thing could work well on a shorter timelibe, especially for new exec?

**Comment by jhogendorn** [comment]
> I have seen other grps have a 12mth handover, so exec are 'incoming' for a year, 'current' for a year, 'outgoing' for a year.. but i suspect that a 3yr commiment is overkill fir hsbne. .. but maybe the overlapping thing could work well on a shorter timelibe, especially for new exec?

Yeah when we did the training courses the council offers, that came up. the apprentice/hold/outward model. its a big commitment.

I would really like to see some kind of 'apprenticeship' or something, or like a 'shadow exec' who help the exec do stuff and you have to be on the shadow one for x time to be eligable for the full exec or something.

I also think we should assess the roles of vice president and patron, help define those roles a bit more? for example, i think one or the other should be the 'volunteer coordinator and culture' person. basically in charge of new member experience, helping the teams with problems, ensuring volunteers are happy, generally focusing on the 'people' part of the space.

another option is that when you get elected you become the apprentice for a year before taking over, but i think that means if a position isnt contested they dont get help which i'm not a fan of.

at a lot of the core of this stuff is some fundamental problems i guess:

* Handover of all the various things is really complicated and hard to do.
* It would be better to have longer terms because it takes a while to 'get good at it'
* Exec need helpers to get a lot of the day to day stuff done
* Training is a fundamental issue at a lot of levels, but to be exec in particular needs more training. Even just practicalities of knowing all the moving parts, how to use the various bits of software etc.
* Everyone tends to drop all problems on the president, so it would be good to denote the vice/patron roles with better responsibilities to lessen that load.

**Comment by jhogendorn** [comment]
a further thought, perhaps vice/patron can take on the job of 'events'. its a big job but someone needs to focus on it.

**Comment by antifoo** [comment]
I have no doubt that definition of other roles is necessary, however it strays from the objective of this change. 
The goal is to have some level of concurrency of roles to allow for a handover to occur. 
If there is better wording or phrasing that can help achieve that I welcome all input, but other amendments are a distraction.
It will take a change in culture to have a '3 year term', one of apprenticeship and two of responsible duties. I hope that 2 months is sufficient, along with other notes I've made for the process, to ensure that each new appointee is sufficiently educated to complete their obligations satisfactorily.

**Comment by jhogendorn** [comment]
> I have no doubt that definition of other roles is necessary, however it strays from the objective of this change. The goal is to have some level of concurrency of roles to allow for a handover to occur...

Yeah got a bit off piste there. I guess its all kind of one enchilada to some extent.

I think it would be good to define: 

* who is 'in charge' during the dual period. ie outgoing pres/tres/sec is in charge until kpis met? or new team immediate and old team become advisors? or each role transitions as it meets the kpis?
* key handover goals that must be met
    * financial year audit < technically should be done by agm but never is.
    * return of association
    * financial systems and anything thats a pain with kyc stuff (bank, paypal, stripe, budgetly, etc)
    * shadowing on x amount of landlord monthly meetings?
    * secretary duties such as x amount of meeting agendas / notes?

I'm flexible obv, just trying to call out the important things.

**Comment by jhogendorn** [comment]
We could also just write it in as 'kpis as detailed by the handover policy' so its not written in stone in the rules.

**Comment by antifoo** [comment]
I'm always in favour of having reference documentation that doesn't require
onerous approvals.

Murray Wayper
*ScapeVelocity.net <http://scapevelocity.net/>*

**Comment by jhogendorn** [comment]
I think this is supersceded by the total constitution rewrite we did. Closing this pr, let me know if this needs revival.

**Pull request closed without merging**